components/pagination/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@hods/pagination",
+  "version": "0.1.8",
+  "description": "Component to allow separating long tables into separate tables",
+  "main": "src/Pagination.tsx",
+  "sass": "assets/Pagination.scss",
+  "publishConfig": {
+    "main": "dist/Pagination.js",
+    "typings": "dist/Pagination.d.ts"
+  },
+  "files": [
+    "/assets",
+    "/dist"
+  ],
+  "scripts": {
+    "test": "jest",
+    "prepublishOnly": "npm run clean && npm run build",
+    "build": "tsc",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo"
+  },
+  "author": "Daniel A.C. Martin <npm@daniel-martin.co.uk> (http://daniel-martin.co.uk/)",
+  "license": "MIT",
+  "keywords": [
+    "react-components"
+  ],
+  "dependencies": {
+    "@hods/sass-base": "^0.1.10",
+    "@not-govuk/component-helpers": "^0.2.9",
+    "@not-govuk/docs-components": "^0.2.9",
```

Comment:
This line shouldn't be here. :-)

I'm fairly sure it's the cause of your missing code snippets.

Basically the (React) snippets are taken out of a React 'context' and React contexts are a nuisance. You need to make sure you share exactly the same module as the docs application so it needs to be a peer dependency.